Libplanet.Store/StoreExtensions.cs

```diff
@@ -36,7 +35,7 @@ public static void Copy(this IStore from, IStore to)
             {
                 foreach (BlockHash blockHash in from.IterateIndexes(chainId))
                 {
-                    Block block = from.GetBlock(blockHash);
+                    Block block = from.GetBlock(blockHash)!;
```

Comment:
Should also throw an `Exception` with an explanation. We might want to add that we expect the source `IStore` to be "complete" in the sense that it contains all `Block`s.

Reply:
How about the code below?
```csharp
Block? block = from.GetBlock(blockHash);
if (block is null)
{
    var message = $"Cannot found block: '{blockHash}' in store.";
    throw new InvalidOperationException(message);
}
```

Reply:
I'd suggest

```csharp
Block block = from.GetBlock(blockHash) ??;
    throw new InvalidOperationException($"Cannot found block: '{blockHash}' in store.");
```

instead.

- It is shorter and easier to comprehend (for me at least)
- `block` isn't really ever supposed to be `null`.